### PR TITLE
Abcjs

### PR DESCRIPTION
**Pull request by kotaro44**

### CODE REVIEW

**Comment by mscuthbert** [comment]
Can you point me to a discussion about this -- what this PR attempts to do, etc.?  Or change the description to be more meaningful.  Thanks!

File: examples/Basic.html [hunk 1]
```diff
@@ -1,41 +1,49 @@
...
+	<script src="../inc/shim/WebAudioAPI.js"></script>
+	<script src="../inc/shim/WebMIDIAPI.js"></script>
+
 	<!-- utils -->
-	<script src="../js/util/dom_request_xhr.js" type="text/javascript"></script>
-	<script src="../js/util/dom_request_script.js" type="text/javascript"></script>
+	<script src="../inc/dom/request_script.js"></script>
+	<script src="../inc/dom/request_xhr.js"></script>
+	<script src="../inc/AudioSupports.js"></script>
+	<script src="../inc/EventEmitter.js"></script>
```
Comment (ewilkison) [inline examples/Basic.html:15]: gtt

File: README.md [hunk 2]
```diff
@@ -11,6 +12,9 @@
 * [Color Piano](http://mudcu.be/piano) by Michael Deal @mudcube
 * [Euphony 3D Piano](http://qiao.github.com/euphony/) by Xueqiao Xu @qiao
 * [Gbloink!](http://gbloink.com/alpha/) by Phil Jones
+
```
Comment (emma-wilkinson) [inline README.md:15]: ffff

File: README.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ## Code examples - from the repo
 
-* [Basic](./examples/Basic.html) - the most basic implementation.
-* [MIDIPlayer](./examples/MIDIPlayer.html) - how to parse MIDI files, and interact with the data stream.
-* [WhitneyMusicBox](./examples/WhitneyMusicBox.html) - a audio/visual experiment by Jim Bumgardner
+* [Basic](https://mudcu.be/github/midi-js/Basic.html) - the most basic implementation.
+* [MIDIPlayer](https://mudcu.be/github/midi-js/MIDIPlayer.html) - how to parse MIDI files, and interact with the data stream.
```
Comment (emma-wilkinson) [inline README.md:4]: multi line example

File: README.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
 ## Code examples - from the repo
 
-* [Basic](./examples/Basic.html) - the most basic implementation.
-* [MIDIPlayer](./examples/MIDIPlayer.html) - how to parse MIDI files, and interact with the data stream.
-* [WhitneyMusicBox](./examples/WhitneyMusicBox.html) - a audio/visual experiment by Jim Bumgardner
```
Comment (emma-wilkinson) [inline README.md:5]: single line

File: README.md [hunk 5]
```diff
@@ -1,8 +1,9 @@
...
+* [Basic](https://mudcu.be/github/midi-js/Basic.html) - the most basic implementation.
+* [MIDIPlayer](https://mudcu.be/github/midi-js/MIDIPlayer.html) - how to parse MIDI files, and interact with the data stream.
+* [MultipleInstruments](https://mudcu.be/github/midi-js/MultipleInstruments.html) - synth drum and piano playing together
+* [WhitneyMusicBox](https://mudcu.be/github/midi-js/WhitneyMusicBox.html) - a audio/visual experiment by Jim Bumgardner
```
Comment (emma-wilkinson) [inline README.md:7]: new comment

**Pull request closed without merging**